engines/director/lingo/lingo.cpp

```diff
@@ -277,7 +278,12 @@ Common::String LingoArchive::formatFunctionList(const char *prefix) {
 		if (scriptContexts[i].size() == 0)
 			result += Common::String::format("%s  [empty]\n", prefix);
 		for (ScriptContextHash::iterator it = scriptContexts[i].begin(); it != scriptContexts[i].end(); ++it) {
-			result += Common::String::format("%s  %d:\n", prefix, it->_key);
+			result += Common::String::format("%s  %d", prefix, it->_key);
```

Comment:
Are two spaces between `%s` and `%d` intentional?

Reply:
Yes, they are for indenting the list items